code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -697,4 +697,31 @@
 	if (.)
 		var/obj/item/borg/apparatus/beaker/extra/E = locate() in R.module.modules
 		if (E)
-			R.module.remove_module(E, TRUE) 
\ No newline at end of file
+			R.module.remove_module(E, TRUE)
+
+/obj/item/borg/upgrade/exp_welder

```

Comment:
Please move this to a new NSV13 file eg nsv13/code/game/objects/items/robot/custom_robot_upgrades.dm or something like that.